pkg/virt-api/validating-webhook/validating-webhook_test.go

```diff
@@ -959,6 +959,81 @@ var _ = Describe("Validating Webhook", func() {
 			}
 		})
 	})
+	Context("with cpu pinning", func() {
+		var vmi *v1.VirtualMachineInstance
+		BeforeEach(func() {
+			vmi = v1.NewMinimalVMI("testvmi")
+			vmi.Spec.Domain.CPU = &v1.CPU{DedicatedCPUPlacement: true}
+		})
+		It("should reject specs without cpu reqirements", func() {
+			causes := ValidateVirtualMachineInstanceSpec(k8sfield.NewPath("fake"), &vmi.Spec)
+			Expect(len(causes)).To(Equal(1))
+			Expect(causes[0].Field).To(Equal("fake.domain.cpu.dedicatedCpuPlacement"))
+		})
+		It("should reject specs without inconsistent cpu reqirements", func() {
+			vmi.Spec.Domain.CPU.Cores = 4
+			vmi.Spec.Domain.Resources.Limits = k8sv1.ResourceList{
+				k8sv1.ResourceCPU: resource.MustParse("2"),
+			}
+			causes := ValidateVirtualMachineInstanceSpec(k8sfield.NewPath("fake"), &vmi.Spec)
+			Expect(len(causes)).To(Equal(1))
+			Expect(causes[0].Field).To(Equal("fake.domain.cpu.dedicatedCpuPlacement"))
+		})
+		It("should reject specs with non-integer cpu limits values", func() {
+			vmi.Spec.Domain.Resources.Limits = k8sv1.ResourceList{
+				k8sv1.ResourceCPU: resource.MustParse("800m"),
```

Comment:
maybe add another negative test with a fraction value (cpu = 1.5) not in milli format.

Reply:
@vladikr if it does so i do :)